src/Http/Controllers/Character/WalletController.php

```diff
@@ -24,7 +24,7 @@
 
```

Comment:
# Character Wallet

## Transaction
- sorting by total is not working properly (is there a missing attribute ?)